### DISCUSSION TITLE
I'm trying to make a game-like educational tool using SpiceSharp for my university project. keep having problems! any help would be appreciated!

### DISCUSSION THREAD

**Discussion by MrMemozz** [SpiceSharp/SpiceSharp#222] [Q&A] [answered]
Hello everyone. As per the title, my main goal is having a semi-polished game-like circuit simulator for my university, but since I'm a beginner level programmer and I'm studying Electrical Engineering so I'm not even majoring in software, i feel like making this program would really boost what I'm missing in software skills.
 this is my first time using GitHub for reasons other than browsing, i have no idea about licensing and open sources and things like that so firstly i have to ask if my project is a non-profit educational tool, and all references will be mentioned, is it okay if i continue with it and submit it as a university project?
 and if that is okay, my goal is for it to be opensource, as in after i finish it i will upload all the scripts to GitHub or whatever platform is suitable and maybe comeback every once in a while to check how real programmers improve the idea hopefully.
 is that also okay?
 and thanks in advance!
 Edit: i forgot to add that if everything is okay, should i ask questions about problems I'm encountering in integrating this to unity in this same thread or do i make new threads so if other people have similar problems in the future to find a solution for them?

**Comment by svenboulanger** [comment]
Hi,

The source is under MIT license, which means you can use and modify the code however you want. Using any substantial parts of the code would require you to give credit. You can check the license if you want as it's included in the repo. MIT license is known for being short and simple.

As far as I'm concerned, there are no real practical limitations to using this library for your project.

Yours sincerely,
Sven

**Comment by svenboulanger** [comment, marked as answer]
If you have questions, please start a discussion. If you think the library is not doing what you believe it should be doing, please post an issue. 🙂

**Comment by MrMemozz** [comment]
Thank you so much Sven !
